src/status_im/contexts/communities/events.cljs

```diff
@@ -34,15 +35,12 @@
       {:db (assoc-in db
             [:communities id]
             (assoc community :last-opened-at (max last-opened-at previous-last-opened-at)))
-       :fx [[:dispatch [:communities/initialize-permission-addresses id]]
```

Comment:
FYI: When a community needs to be reconciled into the app-db, we now don't further compute the revealed accounts and initiate the permissions addresses. Doing this is rather inefficient for a community entity that can change frequently. Instead, these events are called somewhere closer to where the view that needs them will be mounted, using, for example, `rn/use-mount`.